src/zen/common/ZenUIManager.mjs

```diff
@@ -37,29 +37,73 @@ var gZenUIManager = {
     document.addEventListener('mousedown', this.handleMouseDown.bind(this), true);
 
     // Add global click handler for split link feature
-    document.addEventListener('click', (event) => {
+    document.addEventListener('click', async (event) => {
       // Only handle left-clicks
       if (event.button !== 0) return;
       // Only handle anchor elements
       const anchor = event.target.closest('a[href]');
       if (!anchor) return;
       // Ignore if default prevented or not in main content area
       if (event.defaultPrevented) return;
-      // Check for Shift key
-      if (!event.shiftKey) return;
+      // Only trigger if Shift is pressed and no other modifier
+      if (!gZenUIManager._isOnlyShift(event)) return;
+
+      // Check if Glance is configured to use Shift+Click
+      if (typeof gZenUIManager._getGlanceActivationMethod === 'function') {
+        const glanceActivation = await gZenUIManager._getGlanceActivationMethod();
+        if (glanceActivation === 'shift') {
+          // Let Glance handle this, not split view
+          return;
+        }
+      }
+
       // Prevent default navigation
       event.preventDefault();
       event.stopPropagation();
+      // Normalize URL (absolute if relative)
+      const url = gZenUIManager._getAbsoluteUrl(anchor);
       // Open in split tab
       if (window.gZenViewSplitter && typeof window.gZenViewSplitter.splitLinkInNewTab === 'function') {
         // Temporarily set up gContextMenu to provide the link URL for splitLinkInNewTab
         window.gContextMenu = window.gContextMenu || {};
-        window.gContextMenu.linkURL = anchor.href;
+        window.gContextMenu.linkURL = url;
         window.gZenViewSplitter.splitLinkInNewTab();
         // Clean up
         delete window.gContextMenu.linkURL;
       }
     }, true);
+  // Returns a promise for the current Glance activation method, or undefined if not available
+  async _getGlanceActivationMethod() {
```

Comment:
Why are you using glance related stuff here? I thought the PR was for shift+Click a link